src/scicookie/{{cookiecutter.project_slug}}/README.md

```diff
@@ -4,9 +4,10 @@
 {{ cookiecutter.project_short_description }}
 
 {% if is_open_source -%}
-* Free software: {{ cookiecutter.project_license }}
+
+- Free software: {{ cookiecutter.project_license }}
```

Comment:
it is not related to this PR .. and should be treated in a follow up .. 
but we should change from Free software to Software License